common/changes/office-ui-fabric-react/hovercard-fix_2019-06-18-21-46.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "Moved setSate in _setHeightOffsetEveryFrame so it does not run more than necessary and added and if to check if there is a given position and final height, then stop running _updateAsyncPosition",
```

Comment:
nit: comment in here must be what has been fixed in what component with a short sentence. It helps scanning through release notes. Start with something like: `CalloutContent: fixing bug....`. No need implementation details.

Reply:
It should read like release notes. So what behavior difference does the user care about reading?

Reply:
@sopranopillow ping on this!